### DISCUSSION TITLE
Welcome to the Contribute section of Discussions 👋

### DISCUSSION THREAD

**Discussion by marketachalupnikova** [tolgee/tolgee-platform#1502] [Contribute]
For users interested in contributing to Tolgee, we're here to help. Feel free to ask questions, suggest your ideas, offer feedback or post any contributions you've made.

Thank you! 🙌

**Comment by Mayank77maruti** [comment]
When Can we except to get awesome goodies from tolgee..
Just asking out of excitement :)

**Reply by marketachalupnikova** [reply]
I just sent out the form to the contributors. Sorry for the delay :(
